internal/plugin/generate.go

```diff
@@ -130,7 +130,7 @@ func generateTestAllServices(f *protogen.GeneratedFile, files []File) {
 	f.P("func ", funcName, "(t *", t, ",s ", name, ") {")
 	for _, file := range files {
 		for _, service := range file.services {
-			name := "Test" + string(service.service.Desc.Name())
+			name := "test" + string(service.service.Desc.Name())
```

Comment:
If one wants to explicitly only test 1 service how do you go about that with this change? Do you have to use the `TestServices` API and implement every function for every service you don't intend to test and just return nil in every one?

Reply:
Yes, that would be the case then, or use the other test setup.
I think using the service specific tests directly defeats the whole purpose of the new setup, that you should get a error as soon something gets added.

Reply:
Talked about it, and said that we can start with private functions, and make them public later if needed.